core/datainput.gms

```diff
@@ -1383,12 +1383,8 @@ p_macCostSwitch("co2cement_process") =0 ;
 
 *** load econometric emission data
 *** read in p3 and p4
-table p_emineg_econometric(all_regi,all_enty,p)        "parameters for ch4 and n2o emissions from waste baseline and co2 emissions from cement production"
-$ondelim
-$include "./core/input/p_emineg_econometric.cs3r"
```

Comment:
@JakobBD Does this imply that the file can be removed from inputdata generation?

Reply:
Yes. Maybe Jess can confirm, if I remember correctly, it was her adding this, back in the REMIND stone age.